repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -263,7 +324,7 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
     }
 
     it("should initialize the cache when given a config") {
-      ReflectionTestUtils.getField(filter, "cache") should be (null)
+      ReflectionTestUtils.getField(filter, "cache") should be(null)
```

Comment:
In this case, parenthesis shouldn't even be necessary. Just replace `should be` with `shouldBe`.

However, IntelliJ does totally want to "fix" this formatting to what you have here. So this will probably come up again (likely in one of my PRs).